default.nix

```diff
@@ -53,6 +58,6 @@ in
   lib = project.stablecoin.components.library;
   test = project.stablecoin.components.tests.stablecoin-test;
   nettest = project.stablecoin.components.tests.stablecoin-nettest;
-  stablecoin-client = project.stablecoin.components.exes.stablecoin-client;
+  static = project.stablecoin.components.exes.stablecoin-client;
```

Comment:
I'd still call this `stablecoin-client` since its linking way is defined by the argument that comes from the outer world

Reply:
Thanks, done!